src/main/kotlin/com/zepben/evolve/services/network/NetworkServiceComparator.kt

```diff
@@ -753,6 +756,24 @@ class NetworkServiceComparator @JvmOverloads constructor(
             compareConductingEquipment()
         }
 
+    private fun compareGroundingImpedance(source: GroundingImpedance, target: GroundingImpedance): ObjectDifference<GroundingImpedance> =
+        ObjectDifference(source, target).apply {
+            compareEarthFaultCompensator()
+
+            compareValues(
+                GroundingImpedance::x
```

Comment:
todo: single line unless too many to fit in width limit. see other calls in this file. This applies to all calls you have added.

Reply:
changed